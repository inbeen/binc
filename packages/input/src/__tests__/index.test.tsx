import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import { Input, TextArea } from '..';

describe('Input', () => {
  it('should work', () => {
    const { asFragment } = render(<Input />);
    expect(asFragment().firstChild).toMatchSnapshot();
  });
});

describe('Textarea', () => {
  it('should work', () => {
    const { asFragment } = render(<TextArea />);
    expect(asFragment().firstChild).toMatchSnapshot();
  });
});
