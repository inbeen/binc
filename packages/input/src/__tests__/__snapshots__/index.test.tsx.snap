// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > should work 1`] = `
<input
  class="ant-input css-dev-only-do-not-override-1c0na6j ant-input-outlined"
  type="text"
  value=""
/>
`;

exports[`Textarea > should work 1`] = `
<textarea
  class="ant-input css-dev-only-do-not-override-1c0na6j ant-input-outlined"
/>
`;
